Windows build error on linux

```
x86_64-w64-mingw32-gcc   -std=gnu99 -W -Wall -Wextra -O2 -pipe -Iinclude/ -IOpenCL/ -Ideps/LZMA-SDK/C -Ideps/zlib -Ideps/zlib/contrib -Ideps/OpenCL-Headers -DWITH_BRAIN -Ideps/xxHash -DWITH_CUBIN -Ideps/unrar -fPIC -I/opt/win-iconv-64/include/ -DWITH_HWMON      -o hashcat.exe src/main.c obj/combined.WIN.a /opt/win-iconv-64/lib/libiconv.a  -s -lstdc++ -Wl,--dynamicbase -Wl,--nxcompat -lpsapi -lws2_32 -lpowrprof -static -static-libgcc -static-libstdc++   -DCOMPTIME=1648145764 -DVERSION_TAG=\"v6.2.5-358-g84acc32a7\"
src/main.c: In function ‘main_log’:
src/main.c:82:15: warning: unused variable ‘orig’ [-Wunused-variable]
   82 |     const int orig = con_info.wAttributes;
      |               ^~~~
src/main.c:117:55: error: ‘hConsole’ undeclared (first use in this function)
  117 |       case LOGLEVEL_WARNING: SetConsoleTextAttribute (hConsole, orig); break;
      |                                                       ^~~~~~~~
src/main.c:117:55: note: each undeclared identifier is reported only once for each function it appears in
src/main.c:117:65: error: ‘orig’ undeclared (first use in this function)
  117 |       case LOGLEVEL_WARNING: SetConsoleTextAttribute (hConsole, orig); break;
      |                                                                 ^~~~
make: *** [src/Makefile:826: hashcat.exe] Error 1
```

**To Reproduce**
`make win` on Ubuntu or WSL Ubuntu

**Expected behavior**
Build completes

**Hashcat version (please complete the following information):**
 - OS: Linux (on WSL)
 - Distribution: Ubuntu 20.04
 - Version: 20.04.4

**Fix proposal**

```
index 7c77433a7..629f1282c 100644
--- a/src/main.c
+++ b/src/main.c
@@ -70,8 +70,6 @@ static void main_log (hashcat_ctx_t *hashcat_ctx, FILE *fp, const int loglevel)
     event_ctx->prev_len = msg_len;
   }

-  // color stuff pre
-  if (is_stdout_terminal()) {
   #if defined (_WIN)
     HANDLE hConsole = GetStdHandle (STD_OUTPUT_HANDLE);

@@ -80,6 +78,11 @@ static void main_log (hashcat_ctx_t *hashcat_ctx, FILE *fp, const int loglevel)
     GetConsoleScreenBufferInfo (hConsole, &con_info);

     const int orig = con_info.wAttributes;
+  #endif
+
+  // color stuff pre
+  if (is_stdout_terminal()) {
+  #if defined (_WIN)

     switch (loglevel)
     {
```

--- a/src/main.c
+++ b/src/main.c
@@ -70,17 +70,19 @@ static void main_log (hashcat_ctx_t *hashcat_ctx, FILE *fp, const int loglevel)
     event_ctx->prev_len = msg_len;
   }
 
+  #if defined (_WIN)
+  HANDLE hConsole = GetStdHandle (STD_OUTPUT_HANDLE);
+
+  CONSOLE_SCREEN_BUFFER_INFO con_info;
+
+  GetConsoleScreenBufferInfo (hConsole, &con_info);
+
+  const int orig = con_info.wAttributes;
+  #endif
+
   // color stuff pre
   if (is_stdout_terminal()) {
   #if defined (_WIN)
-    HANDLE hConsole = GetStdHandle (STD_OUTPUT_HANDLE);
-
-    CONSOLE_SCREEN_BUFFER_INFO con_info;
-
-    GetConsoleScreenBufferInfo (hConsole, &con_info);
-
-    const int orig = con_info.wAttributes;
-
     switch (loglevel)
     {
       case LOGLEVEL_INFO:

